Moonshot/player/player.gd

```diff
@@ -38,21 +38,28 @@ var safety = false
 var dead = false
 var priority_animations = ["stagger", "dodge"]
 
+var mdiff : Vector2
+var mouse : Vector2
+var msspd : float
+#var pos: Vector2
+var joys : Vector2
```

Comment:
What is `joys` ?

Reply:
tracks joystick position.
But did you test this? i wasnt putting this up for a code quality check